docs/05-concepts/06-database/01-connection.md

```diff
@@ -62,3 +70,59 @@ To remove the database and __delete__ all data add the `-v` flag.
 $ docker compose down -v
 ```
 
+
+## Connecting to a Postgresql Server Instance
+
+Connecting to a Postgresql Server Instance is similar to connecting to a Docker Image, but there are some prerequisites to take into account:
+
+- Making sure that the machine/server allows incoming traffic
+- Creating a Postgresql user and database
+
+You can connect to Postgresql server that you have installed on your local machine, a self-hosted one, Google Cloud SQL, AWS RDS or SQL Server, or any other 3rd party service.
```

Comment:
Suggestion: Improve the start of the sentence and clarify that any service is supported.

The start of this sentence seems a little strange. How about:

Serverpod supports connecting to any running Postgresql server, whether hosted on your local machine in Google Cloud SQL or any other 3rd party service.

Reply:
Maybe this sentence should be directly under the `Connecting to a Postgresql Server Instance` header as well, and then we list the prerequisites below. "Sales pitch" before instructions?